### PR TITLE
i18n(ko-KR): update `content-collections.mdx`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### Description (required)

- update `content-collections.mdx`
- #8967
- #8988

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

File: src/content/docs/ko/guides/content-collections.mdx [hunk 1]
```diff
@@ -535,6 +535,37 @@ const { Content } = await entry.render();
  
 [블로그 만들기 튜토리얼의 완성된 프로젝트](https://github.com/withastro/blog-tutorial-demo)의 코드베이스를 사용하는 [단계별 튜토리얼](/ko/tutorials/add-content-collections/)에서 기본 블로그 예시를 `src/pages/posts/` 디렉터리에서 `src/content/posts` 디렉터리로 변환하는 방법을 알아보세요.
 
+## JSON 스키마 생성 활성화
+
+<p><Since v="4.13.0" /></p>
+
+`data` 타입의 컬렉션으로 작업하는 경우, Astro는 편집기에서 자동 완성 및 타입 검사를 수행할 수 있도록 JSON 스키마 파일을 자동으로 생성합니다. [`zod-to-json-schema`](https://github.com/StefanTerdell/zod-to-json-schema#known-issues)라는 라이브러리를 사용하여 `src/content/config.ts`에 정의된 컬렉션을 기반으로 프로젝트의 각 데이터 컬렉션에 대한 별도의 파일이 만들어집니다.
+
+이 기능을 사용하려면 컬렉션의 각 데이터 엔트리 파일에서 `$schema`의 값을 수동으로 스키마 파일의 경로로 설정해야 합니다:
```
Comment (dreyfus92) [inline src/content/docs/ko/guides/content-collections.mdx:544]:
```suggestion
이 기능을 사용하려면 컬렉션의 각 데이터 엔트리 파일에서 `$schema` 의 값을 수동으로 스키마 파일의 경로로 설정해야 합니다:
```

**Review by dreyfus92 — COMMENTED** [review]
Awesome work, just a nit 😁

File: src/content/docs/ko/guides/content-collections.mdx (hunk 1 above)
Comment (jsparkdev) [reply src/content/docs/ko/guides/content-collections.mdx:544]: Thank you! But in this case, it is more natural to use the two characters together without spacing.

**Review by yanthomasdev — APPROVED** [review]
LGTM! Good work! 🙌

**Pull request merged**